src/mats_l1_processing/read_parquet_functions.py

```diff
@@ -227,10 +270,23 @@ def read_ccd_data_in_interval(
                 & (ds.field(variable) <= filter[variable][1])
             )
 
+    st = time.time()
+    table = dataset.to_table(
+        filter=partition_filter & filterlist
+    )
+    et = time.time()
+    print('Execution time:', et - st, 'seconds' )
+    print(len(table))
```

Comment:
```suggestion
    print(f"Got {len(table)} records {time.time() - st} seconds'")
```
or something like that, if you want to keep it, but are these print outs meant to be there, or are they for debugging? I don't like unsolicited output, so I would have liked to see this controlled by some sort of semaphore, in that case, but that's not the important bit.

Reply:
(Note: Same thing further down.)

Reply:
Yes these timings should be removed (fixed now in latest patch)